meta-resin-common/recipes-core/dropbear/dropbear_%.bbappend

```diff
@@ -29,18 +29,16 @@ do_install_append() {
         echo 'DROPBEAR_EXTRA_ARGS="-s"' >> ${D}/etc/default/dropbear
     fi
 
-    if [ "${RESIN_CONNECTABLE_ENABLE_SERVICES}" = "1" ]; then
```

Comment:
That is a bad change for all custom builds like hass.io and also for idea to sync authorized_keys with a boot file.